lesson17/script.js

```diff
@@ -0,0 +1,147 @@
+const slider = document.getElementById("js-slider");
+
+const createElement = ({tagName,id,className,text}) => {
+    const element = document.createElement(tagName);
+    if(id){
+        element.id = id;
+    }
+    element.className = className;
+    element.textContent = text;
+    return element;
+}
```

Comment:
I like this function!
Good idea. I want to copy it too.👏

Q
`createImage` and  `createButton` 
I thought the code was a bit redundant as these functions are similar in processing to this one. Can you make it a little more generic?

Reply:
Thank you for your review!
Please copy and use it:smile:

A
That's true!
I will try it generic.

Reply:
I fixed.
https://github.com/nakagawamai/js-lesson/pull/19/commits/8186edc07df17ed55ff8293414f624f59a50c402